framework/components/ecl-messages/ecl-messages.twig

```diff
@@ -1,5 +1,15 @@
-<div class="messages alert {{messages_type}}">
-  <a href="#" class="close" data-dismiss="alert">&times;</a>
-  <h3><span class="ecl-sr-only">{{messages_sr_only}}:</span>{{messages_title}}</h3>
-  <p>{{messages_data}}</p>
+<div class="ecl-message {% if modifier is defined %} ecl-message--{{ modifier }} {% endif %}">
```

Comment:
I'm wondering if it could be considered as a `dialog`... https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_dialog_role